agents/src/tokenize/basic/sentence.ts

```diff
@@ -67,3 +76,7 @@ export const splitSentences = (text: string, minLength = 20): [string, number, n
 
   return sentences;
 };
+
+const rawRegex = (r: RegExp): string => {
```

Comment:
wouldn't this be the same as `r.source` ?

Reply:
i knew there was a thing for this but i couldn't find it for the life of me :woman_facepalming: